src/routes/metrics/+server.ts

```diff
@@ -0,0 +1,9 @@
+import { register } from "$lib/server/metrics";
+
+export async function GET() {
```

Comment:
Can we expose this route on another port? 
It would be better if we can avoid exposing metrics publicly